resources/public/api/conf/1.0/application.yaml

```diff
@@ -12,170 +12,304 @@ This API allows online cashless payment providers for schools to offer their use
+                      description: The sum of paid_in_by_you and government_top_up, this is the total amount paid into the account.
+                    cleared_funds:
+                      type: string
+                      description: The total amount of money available to the parent or guardian to use to make payments to childcare providers. This is equal to total_balance minus any uncleared funds from paid_in_by_you.
+                    top_up_allowance:
+                      type: string
+                      description: The remaining amount of government top-up that is available to the parent or guardian for the current quarter. The amount that is paid in to the account by the government depends on what the parent or guardian pays in.
+          "400":
+            description: Bad request
+            content:
+                application/json:
+                  schema:
+                    $ref: '#/components/schemas/errorResponse'
+                  examples:
+                    invalid_input_data:
+                      value:
+                        errorDescription: The data entered is invalid
+                    missing_correlation_id:
+                      value:
+                        errorDescription: correlation_id is missing
+                    missing_epp_unique_customer_id:
+                      value:
+                        errorDescription: "epp_unique_customer_id is missing"
+                    missing_epp_reg_reference:
+                      value:
+                        errorDescription: "epp_reg_reference is missing"
+                    missing_outbound_child_payment_ref:
+                      value:
+                        errorDescription: "outbound_child_payment_ref is missing"
+                    missing_child_date_of_birth:
+                      value:
+                        errorDescription: "child_date_of_birth is missing"
+                    invalid_child_date_of_birth:
+                      value:
+                        errorDescription: "child_date_of_birth does not match the expected format (YYYY-MM-DD)"
+                    epp_unique_customer_id_does_not_match_epp_reg_reference:
+                      value:
+                        errorDescription: "epp_unique_customer_id does not correlate with the provided epp_reg_reference"
+                    child_date_of_birth_does_not_match_outbound_child_payment_ref:
+                      value:
+                        errorDescription: "child_date_of_birth does not correlate with the provided outbound_child_payment_ref"
+          "401":
+            description: Unauthorized
+            content:
+                application/json:
+                  schema:
+                    $ref: '#/components/schemas/errorResponse'
+                  examples:
+                    invalid_credentials:
+                      value:
+                        errorDescription: "Invalid OAuth token supplied for user-restricted endpoint"
```

Comment:
401 isn't only returned when the bearer token is invalid. It could also be returned due to the `Authorization` header missing entirely or the token bearer not satisfying the predicate supplied to Auth (which in our case is requiring a confidence level of 250).

Reply:
I have added a message for a missing header, however I believe the second case is covered by the original `invalid_credentials` example